### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by Leboroz**
# BallheadApp [LIVE](https://leboroz-ballhead.netlify.com)

finish

## Learning objectives

- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.

>This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I created making use of React and Redux.

### pages: 

- one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country.

- one page for the item details; in the example, the detail page for Czech Republic cities with a number of views.

#### Home page

- When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
- Along with the category name, you will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.

#### Details page

- In the details page, the webapp retrieves data from the API to show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.

### CODE REVIEW

File: src/pages/About.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        return (
+          <Card
+            key={name}
+            name={name}
+            value={pollutant[1]}
+            even={index % 2 !== 0}
+          />
+        );
+      })) || <h1>loading</h1>
+  );
```
Comment (bimbolabuari) [inline src/pages/About.jsx:76]: Currently, the detail page isn't showing the country statistics as required.

image

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Leboroz, 

You have done a good job so far 🚀  

## Highlights 🎉 
- Gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
- Deploys application using Netlify
- The project contains a `Home` page and a `Detail` page

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
- [x] Kindly add a video presentation that contains a demo of the project, highlighting all the project features

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: README.md [hunk 2]
```diff
@@ -1,72 +1,107 @@
...
+  - React
+  - redux
 - Technologies used
+  - NPM
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://leboroz-pollution.netlify.app)
```
Comment (moise10r) [inline README.md]: - [ ] Kindly, Update the live demo link in your README file the one you have does not work

File: src/pages/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+import WorldMap from 'react-svg-worldmap';
+import Grid from './Grid';
+import About from './About';
+import Navbar from '../components/Navbar';
+import styles from '../sass/pages/home.module.scss';
+
+const Home = () => {
+  const [header, setHeader] = useState('STATS BY COUNTRY');
+  const [menu, setMenu] = useState('bars');
+  const [open, setOpen] = useState(false);
```
Comment (moise10r) [inline src/pages/Home.jsx:12]: - [OPTIONAL] Kindly, you should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), what if you move your title or name of the country to the left of your header section as show bellow :arrow_down: 

||  Original [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))  ||
|-|-----------------------------------|-|
||go1||

||||

**Review by moise10r — CHANGES_REQUESTED** [review]
Hi @Leboroz,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- PR has a good title and summary. ✔️
- Professional README file. :heavy_check_mark:
- Good presentation video. ✔️ 
- Your app has been well deployed  ✔️ 
- Your app works as expected  I really appreciate +1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @Leboroz 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**